src/Ledger/LedgerDetails/LedgerRolloverProgress/LedgerRolloverProgress.test.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { IntlProvider } from 'react-intl';
+import { MemoryRouter } from 'react-router-dom';
+
+import '@folio/stripes-acq-components/test/jest/__mock__';
+
+import {
+  OVERALL_ROLLOVER_STATUS,
+} from '../../../common/const';
+import LedgerRolloverProgress from './LedgerRolloverProgress';
+
+const LEDGER_NAME = 'test ledger';
+
+const renderComponent = ({
+  ledgerName,
+  fromYearCode,
+  rolloverStatus,
+  onClose = () => { },
+}) => (render(
+  <IntlProvider locale="en">
+    <MemoryRouter>
+      <LedgerRolloverProgress
+        fromYearCode={fromYearCode}
+        ledgerName={ledgerName}
+        onClose={onClose}
+        rolloverStatus={rolloverStatus}
+      />
+    </MemoryRouter>
+  </IntlProvider>,
+));
+
+describe('LedgerRolloverProgress', () => {
+  it('should display ledger name', () => {
+    renderComponent({ ledgerName: LEDGER_NAME });
+    expect(screen.getByText(LEDGER_NAME)).toBeDefined();
+  });
+
+  it('should display progress bar', () => {
+    const { container } = renderComponent({
+      rolloverStatus: { budgetsClosingRolloverStatus: OVERALL_ROLLOVER_STATUS.inProgress },
+    });
+
+    expect(container.getElementsByClassName('progressCurrent')[0]).toHaveStyle('width: 33%');
```

Comment:
why 33%? )

Reply:
this is behavior of Progress component, kind of integration test)

Reply:
I mean, it's not so obvious (for me as a new reader of this code), and some kind of magic

Reply:
changed to simple checking if it's displayed